.github/workflows/builds.yml

```diff
@@ -280,6 +280,8 @@ jobs:
         run: |
           brew install ninja
 
+      - uses: Bacondish2023/setup-googletest@v1
```

Comment:
Remove this

Reply:
Then it doesn't find gmock (see https://github.com/input-leap/input-leap/actions/runs/11175658238/job/31067450751?pr=1984)